src/olympia/blocklist/tests/test_mlbf.py

```diff
@@ -86,32 +91,33 @@ def test_invalid_key_access_raises(self):
         with self.assertRaises(AttributeError):
             loader['invalid']
         with self.assertRaises(AttributeError):
-            loader[BlockType.BLOCKED]
+            loader[BlockType.BANANA]
 
     def test_valid_key_access_returns_expected_data(self):
         loader = self.TestStaticLoader(self.storage)
-        assert loader[MLBFDataType.HARD_BLOCKED] == ['blocked:version']
+        assert loader[MLBFDataType.BLOCKED] == ['blocked:version']
         assert loader[MLBFDataType.NOT_BLOCKED] == ['notblocked:version']
 
     def test_cache_raw_data(self):
         loader = self.TestStaticLoader(self.storage)
 
         for data_type in MLBFDataType:
-            assert loader[data_type] == loader._raw[data_type.value]
+            assert loader[data_type] == loader._raw[loader.data_type_key(data_type)]
 
         # The source of truth should ultimately be the named cached properties
         # Even though _raw is cached, it should still return
         # the reference to the named property
         loader.blocked_items = []
-        assert loader[MLBFDataType.HARD_BLOCKED] == []
+        assert loader[MLBFDataType.BLOCKED] == []
 
 
 class TestMLBFStorageLoader(_MLBFBase):
     def setUp(self):
         super().setUp()
         self._data = {
-            'blocked': ['blocked:version'],
-            'not_blocked': ['notblocked:version'],
+            'blocked_items': ['blocked:version'],
+            'not_blocked_items': ['notblocked:version'],
+            'soft_blocked_items': ['softblocked:version'],
```

Comment:
To me, this means the already persisted data won't be compatible anymore, likely leading to an error. Did you try to run the code locally to see how that would behave with cached data?

Reply:
I'm not sure I follow.. can you clarify what "this" is in the original statement. And what would the "already persisted data" not be compatible with? What kind of error would you expect? What behaviour am I testing against cached data. And by cached data you mean data generated from an existing filter using the storage loader class right?

Reply:
The diff.

Reply:
Now I get it. Compatible as in the name of the keys is not consistent with the already persised cache.json keys "blocked" != "blocked_items". Fixed.